test/functional/outputs/forward_to_cloudwatch_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"crypto/tls"
 	"encoding/json"
+	runtime2 "github.com/openshift/cluster-logging-operator/internal/runtime"
```

Comment:
Why runtime2 and not runtime?

Reply:
Updated to:
* remove ref to `runtime2`
* use `testruntime` where there was conflict